src/lib/js-source-map.ts

```diff
@@ -0,0 +1,45 @@
+import * as sourcemap from 'source-map'
+import {MappingItem} from 'source-map'
+
+type JavascriptSymbolMap = Map<number, Map<number, string>>
```

Comment:
Super nitty nit, but `JavaScriptSymbolMap` please, and similarly `importJavaScriptSymbolMap`.